pagopa/io-app
# [#172240705] Add donations to app

**Short description:**
Include a summary of the changes.

**List of changes proposed in this pull request:**
- add donations screens:
    - DonationsHomeScreen: contains the list of available donation entities
    - DonationDetailScreen: here the user can view the donation video and details
<img width="866" alt="image" src="https://user-images.githubusercontent.com/38431762/79581295-87e67d00-80ca-11ea-89d8-182b7f15253e.png">


**How to test:**
Add the config variable DISPLAY_DONATIONS=YES


TODO:
- remove mocked donations data and add request to the backend to: get the list of available donations, generate payment identification

- solve TODO expressed in the code: they are related to check the custom amount (eg. it has to be < 100 euros), connect DonationDetailScreen to payment flux
